app/components/user_row/user_row.html.erb

```diff
@@ -11,7 +11,7 @@
       <span class="UserRow-name"><%= user.name %></span>
     </strong>
 
-    <span class="UserRow-channels">
+    <span class="<%= channel_css_classes %>">
```

Comment:
What do you think, necessary to write a test for that? The entire `user_row` component is untested :shrug: